server/backend/local-dev.env

```diff
@@ -10,4 +10,6 @@ COGNITO_USER_POOL_DOMAIN=dev-fam-user-pool-domain
 ENABLE_BCSC_JWKS_ENDPOINT=False
 BCSC_KEY_ID=thisisasecret
 FC_API_TOKEN=thisisasecret
-DB_STRING_OVERRIDE=postgresql+psycopg2://fam_proxy_api:test@localhost:5432/fam
\ No newline at end of file
+DB_STRING_OVERRIDE=postgresql+psycopg2://fam_proxy_api:test@localhost:5432/fam
+IDIM_PROXY_BASE_URL=https://nr-fam-idim-lookup-proxy-test-backend.apps.silver.devops.gov.bc.ca
+IDIM_PROXY_API_KEY=thisisasecret
```

Comment:
We probably need to update a developer readme for instructions on how to set up their local environment from scratch.

Reply:
New instruction added in Readme. Please see [here](https://github.com/bcgov/nr-forests-access-management/pull/728/files#diff-a3f3994eac52b3c914f626143baf4e4d2a3408088d077412ce005fcb0ee1a226). Thanks.